com.unity.render-pipelines.high-definition/Runtime/Lighting/Shadow/ScreenSpaceShadowManager.RenderGraph.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Experimental.Rendering.RenderGraphModule;
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+    public partial class HDRenderPipeline
+    {
+        public TextureHandle CreateScreenSpaceShadowTextureArray(RenderGraph renderGraph)
+        {
+            int numShadowTextures = Math.Max((int)Math.Ceiling(m_Asset.currentPlatformRenderPipelineSettings.hdShadowInitParams.maxScreenSpaceShadowSlots / 4.0f), 1);
+            GraphicsFormat graphicsFormat = (GraphicsFormat)m_Asset.currentPlatformRenderPipelineSettings.hdShadowInitParams.screenSpaceShadowBufferFormat;
+            return renderGraph.CreateTexture(new TextureDesc(Vector2.one, true, true)
+            {
+                colorFormat = graphicsFormat,
+                slices = numShadowTextures * TextureXR.slices,
+                dimension = TextureDimension.Tex2DArray,
+                filterMode = FilterMode.Point,
+                enableRandomWrite = true,
+                useDynamicScale = true,
```

Comment:
useDynamicScale is already set by the first param of the TextureDesc constructor so you can remove it as it is redundant in this case.